src/universal/components/AsyncRoute/AsyncRoute.js

```diff
@@ -3,18 +3,29 @@ import {Route} from 'react-router-dom';
 import PropTypes from 'prop-types';
 import Bundle from '../Bundle/Bundle';
 
-const AsyncRoute = ({mod, exact, path, isPrivate, ...extra}) => (
-  <Route
-    exact={exact} path={path}
-    render={({history, location, match}) => (
-      <Bundle extra={extra} history={history} isPrivate={isPrivate} location={location} match={match} mod={mod} />
+const AsyncRoute = ({mod, exact, path, isPrivate, bottom, extraProps}) => {
+  return (
+    <Route
+      exact={exact} path={path}
+      render={({history, location, match}) => (
+        <Bundle
+          bottom={bottom}
+          extraProps={extraProps}
+          history={history}
+          isPrivate={isPrivate}
+          location={location}
+          match={match}
+          mod={mod}
+        />
     )}
-  />
-);
+    />
+  );
+};
 
 AsyncRoute.propTypes = {
+  bottom: PropTypes.bool,
```

Comment:
I'm racking my brain to think of a more descriptive name than bottom. Something that wraps the context that "I'm involved in routing" while still being descriptive. Perhaps, `leafRoute` would be better?